input/groups/armor.json

```diff
@@ -1826,32 +1776,87 @@
     },
     "Guild Wars 2: Secrets of the Obscure (SotO)": {
       "Astral Ward": [
-        11580, 11585, 11591, 11593, 11628, 11637, 11687, 11688, 11692, 11694, 11712, 11722, 11732, 11742, 11751, 11755, 11775, 11786
+        11580,
+        11585,
+        11591,
+        11593,
+        11628,
+        11637,
+        11687,
+        11688,
+        11692,
+        11694,
+        11712,
+        11722,
+        11732,
+        11742,
+        11751,
+        11755,
+        11775,
+        11786
+      ],
+      "Obsidian": [
       ],
       "Rift Hunter": [
-        11572, 11586, 11606, 11630, 11638, 11653, 11657, 11665, 11681, 11684, 11740, 11759, 11763, 11788, 11789, 11806, 11812, 11816
+        11572,
+        11586,
+        11606,
+        11630,
+        11638,
+        11653,
+        11657,
+        11665,
+        11681,
+        11684,
+        11740,
+        11759,
+        11763,
+        11788,
+        11789,
+        11806,
+        11812,
+        11816
       ],
       "Sanctified": [
-        11643, 11654, 11667, 11713, 11727, 11749, 11767, 11777, 11796
+        11578,
+        11611,
+        11629,
+        11643,
+        11654,
+        11667,
+        11682,
+        11689,
+        11709,
+        11713,
+        11727,
+        11749,
+        11752,
+        11764,
+        11767,
+        11777,
+        11793,
+        11796
       ],
       "Standalone: Other": [
-        11569, 11658, 11714
+        11569,
+        11658,
+        11714
       ]
     },
     "Raid": {
       "Dhuum": [
-        7872,
-        7905,
-        7889,
-        7874,
-        7911,
-        7895,
         7848,
-        7868,
         7854,
         7855,
+        7868,
         7871,
-        7887
```

Comment:
If they're sorted numerically then yeah, automation. Ideally everything should be sorted, but it would completely destroy history revision, so not a good idea, at least for now. Should be a once and done thing.

Reply:
As with the formatting conversation, I will run put something together to get all the numbers sorted correctly.